package.json

```diff
@@ -41,7 +41,7 @@
     "prettier": "3.2.5"
   },
   "engines": {
-    "node": ">=14.0.0"
-  },
-  "packageManager": "pnpm@7.5.0"
+    "node": ">=14.0.0",
+    "pnpm": ">=7.5.0"
```

Comment:
I searched a bit since I haven't seen pnpm being used here before, I found a few issues saying that this isn't supported, is it possible for you to see if it works right now, maybe try installing an older version of pnpm in a codespace and try pnpm i

Reply:
Will try.

Reply:
Seems to be working fine.

![image](https://github.com/sliit-foss/npm-catalogue/assets/68442470/6a1db1bd-3d36-4ee3-afb4-32422d759999)